ruby_on_rails/README.md

```diff
@@ -0,0 +1,60 @@
+## Ruby On Rails 5.1
+
+This setup will cover a pure, monolithic, Rails Applications.
+This is the most frequent type of application we have at Renuo and probably also the easiest to setup.
+The application (and relative github repo) will be named after the `[project-name]` you chose before.
+
+**Be careful: this guide explains how to configure a Rails Application and not a Rails API Application!**
+If you have to setup an API-Only application follow the [relative guide](../ruby_on_rails_api/README.md).
```

Comment:
👍  i like it. maybe we could have that for every type: that we explain where to use it and where to not use it, so i can read the first lines and see if i'm on the right track